Dockerfile.windows

```diff
@@ -2,12 +2,14 @@ ARG IMAGE_NAME=newrelic/infrastructure
 ARG IMAGE_TAG=1.10.36-windows-1809-alpha
 FROM $IMAGE_NAME:$IMAGE_TAG
 
+LABEL unsupported=true
```

Comment:
🤔

Reply:
Just to set expectations on people browsing this repo :)